src/commands/new.ts

```diff
@@ -389,30 +391,26 @@ export default {
       overwrite,
     })
     stopSpinner(" 3D-printing a new React Native app", "🖨")
+    // copy the .gitignore if it wasn't copied over
+    // Release Ignite installs have the boilerplate's .gitignore in .gitignore.template
+    // (see https://github.com/npm/npm/issues/3763); development Ignite still
+    // has it in .gitignore. Copy it from one or the other.
+    const boilerplateIgnorePath = exists(boilerplate(".gitignore.template"))
+      ? boilerplate(".gitignore.template")
+      : boilerplate(".gitignore")
```

Comment:
Simplified this logic a little. The `.gitignore.template` should always be there in a production build

However, in development, when running `npx ignite-cli` after running `npx link`, there is chance that `.gitignore.template` will not be there after the `yarn clean` command.